data/ai/lua/ai_helper.lua

```diff
@@ -516,7 +516,10 @@ end
 function ai_helper.split(str, sep)
     -- Split string @str into a table using the delimiter @sep (default: ',')
 
-    local sep, fields = sep or ",", {}
```

Comment:
Is the issue here really the `or` or that it is done together with the comma operator?

Reply:
The issue was that it was declaring `local sep` which shadowed the `sep` function argument.

Reply:
I see. Can't say the new code is more readable personally.

Since `sep` is only used once I'd just put the `or` there. Or if you want to keep the assignment just don't use local?

Reply:
Alright, changed back to being a single line.